src/AppTheme.tsx

```diff
@@ -89,7 +89,7 @@ const themeSettings = createTheme({
                 //   https://github.com/marmelab/react-admin/blob/master/packages/ra-ui-materialui/src/defaultTheme.ts
                 disableRipple: true,
                 sx: {
-                    '&:hover:active::after': {
+                    '&.Mui-focusVisible::after, &:hover::after': {
```

Comment:
The MUI equivalent of the CSS pseudo-class _focus-visible_ was used here, as recommended by the [MUI documentation](https://mui.com/customization/how-to-customize/#state-classes).

Reply:
Good catch!